client/app/components/accounts/wizard/creation.cjsx

```diff
@@ -37,67 +29,21 @@ module.exports = AccountWizardCreation = React.createClass
     ]
 
 
-    # Build state form RequestsStore:
-    # - is an autodiscover request in action, or a provider available?
+    # Build state from RequestsStore through RouterGetter
     getStateFromStores: ->
-        state       = {}
-        discoverReq = RouterGetter.getRequestStatus Requests.DISCOVER_ACCOUNT
-        checkReq    = RouterGetter.getRequestStatus Requests.CHECK_ACCOUNT
-        createReq   = RouterGetter.getRequestStatus Requests.ADD_ACCOUNT
-
-        isBusy = RequestStatus.INFLIGHT in [
-            discoverReq.status
-            checkReq.status
-            createReq.status
-        ]
-
-        # Account creation step
-        if createReq.status is RequestStatus.SUCCESS
-            _.extend state,
-                success: _.partial @redirect, createReq.res
-
-        else if createReq.status is RequestStatus.ERROR
-            _.extend state,
-                alert: ALERTS.CREATE_FAILED
-
-        # Check account failed
-        # - set error message
-        # - if domain is one of known OAuth-aware domain, display reminder about
-        # OAuth token
-        if checkReq.status is RequestStatus.ERROR
-            {oauth} = checkReq.res
-            _.extend state,
-                alert: ALERTS.CHECK_FAILED
-                OAuth: oauth
-
-        # Autodiscover have returned a provider informations (an array of
-        # settings): extract settings from given provider for IMAP and SMTP
-        # and fill state
-        if discoverReq.status is RequestStatus.SUCCESS
-            _.extend state,
-                isDiscoverable: true
-                @parseProviders discoverReq.res
-
-        # autodiscover failed : switch to manual config and set an alert only if
-        # checkReq.status isn't already performed
-        else if discoverReq.status is RequestStatus.ERROR and
-        checkReq.status is null
-            _.extend state,
-                alert: ALERTS.DISCOVER_FAILED
-                isDiscoverable: false
-
-
-        # returns state with its fallback default values.
-        #
-        # `isDiscoverable` is true by default since no check has been performed
-        # (else, it means a previous config - discovered or manual - was
-        # submitted and discovering must be manually set by closing advanced
-        # settings panel)
-        _.defaults state,
-            isBusy:         isBusy
-            isDiscoverable: checkReq.status is null
-            alert:          null
-            success:        false
+        account  = RouterGetter.getAccountCreationSuccess()?.account
+        discover = RouterGetter.getAccountCreationDiscover()
+
+        state =
+            isBusy:         RouterGetter.getAccountCreationBusy()
```

Comment:
`isAccountCreationBusy()` (as what we wantr to known is "are the creation process spawned in background)?